plugin/core/windows.py

```diff
@@ -439,6 +439,7 @@ def update_diagnostics_panel_async(self) -> None:
                 if offset is not None and code is not None and href is not None:
                     prephantoms.append((row, offset, code, href))
                 row += content.count("\n") + 1
+            to_render.append("")  # add spacing between filenames
```

Comment:
Does this require another `row += 1` ?

Reply:
You can check this by running with e.g. eslint and creating diagnostics in at least two files.

Reply:
```diff
 to_render.append("")  # add spacing between filenames
+ row += 1
```
I don't see a difference.
with `row += 1`:

![with](https://user-images.githubusercontent.com/22029477/102546517-0cedc000-40b8-11eb-8d6e-d06b9c372fc4.png)
or without `row += 1`:
![without](https://user-images.githubusercontent.com/22029477/102546523-0eb78380-40b8-11eb-82c1-401df75e430c.png)

But I have noticed on the first line that `typescript:6133` is not highlighted. (I am not a ST syntax expert, not sure how to fix it)
```
     1:1   hint    'styles' is declared but its value is never read. ​typescript:6133
````

Reply:
I'm talking about the clickable links, make sure to check those:

![image](https://user-images.githubusercontent.com/2431823/102550092-863be180-40bd-11eb-82b5-c4722d23ce78.png)

Reply:
Thanks for pointing this out.

> Does this require another row += 1 ?

It turns out that it does: 

![Screenshot from 2020-12-18 21-21-23](https://user-images.githubusercontent.com/22029477/102658269-8993a380-4177-11eb-8fcb-b34c0543f053.png)
the phantom placement is messed up.

It is fixed with 0bcd931

Reply:
> But I have noticed on the first line that `typescript:6133` is not highlighted. (I am not a ST syntax expert, not sure how to fix it)

There is missing `pop: true` in here: https://github.com/sublimelsp/LSP/blob/0bcd93110d98f74879f274b2cb10fa50aec6a663/Syntaxes/Diagnostics.sublime-syntax#L54-L55

Reply:
Thanks @rchl,
I included the fix e729f45